### ISSUE TITLE
How you can change patience value when continuous training yolov8

### ISSUE THREAD

**Issue by itmr97** [ultralytics/ultralytics#3892]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I was training the dataset for 300 epoch and set the value of patience =50 so the training stop at the 183 because  no improvement observed in last 50 epochs so my question How you can change patience value to 100 in continuous training ?

![Screenshot 2023-07-22 205447](https://github.com/ultralytics/ultralytics/assets/40148242/95f7048f-3b5f-4bd8-bd65-14bae3124f1b)

**Comment by glenn-jocher** [comment]
@itmr97 hello,

Thank you for bringing this query to our attention.

When you use the 'patience' value, you intend to halt training if no improvement in loss is observed for a certain number of epochs, in your case, this was initially set to 50. The model stops training at epoch 183 as there is no loss improvement observed in the last 50 epochs.

To modify the 'patience' value for continuous training, you need to adjust the 'patience' parameter present in your script or command where you initiate the model training. Please change this value to 100 and restart the training. The 'patience' setting is not designed to be changed in-between a running training session since the training stages are designed to work with a pre-initialized 'patience' value.

However, it's important to note that increasing the patience value will mean that the model trains for more epochs before halting for lack of improvement, which could also increase the chance of overfitting the model to your training data if not monitored carefully.

I recommend understanding your training data's variance and adjusting your patience value accordingly to strike a balance between training time and model performance.

Thank you for your contribution to the YOLOv8 community and for using our Ultralytics repo.

Best Regards,
Glenn Jocher

**Comment by Ana1890** [comment]
Hi! I have a question about the variable the early stopping is monitoring, it is about the **train box loss** or the **val box loss**?

**Comment by Ana1890** [comment]
> @Ana1890 hi there! 👋 The early stopping feature in YOLOv8 monitors the **val box loss** to decide when to halt training. This helps ensure the model doesn't overfit to the training data and generalizes well to unseen data. Hope this clears things up! If you have any more questions, feel free to ask. Happy training! 🚀

Hi, thanks a lot for the reply. It's just that i don't understand  why my model is keeping training despite I set a petience of 8. Here you can check: 

![image](https://github.com/ultralytics/ultralytics/assets/67620315/358def85-89f8-4eeb-84fb-5f8b033c9fbc)
